.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,18 +5,18 @@ much about the checklist - we will help you get started.
 
 ## Contribution checklist:
 
-(also see [CONTRIBUTING.rst](/CONTRIBUTING.rst) for details)
+(also see [CONTRIBUTING.rst](../tree/master/CONTRIBUTING.rst) for details)
```

Comment:
seems `../CONTRIBUTING.rst` works as well (I just tried it by editing your PR)

Reply:
Interesting... I'm not sure how.

Reply:
https://github.com/tox-dev/tox/CONTRIBUTING.rst `404 Not Found for` me.

`../CONTRIBUTING.rst` is relative to the current page `https://github.com/tox-dev/tox/pull/1363` -> `https://github.com/tox-dev/tox/pull/1363/../CONTRIBUTING.rst` -> `https://github.com/tox-dev/tox/CONTRIBUTING.rst`

Reply:
iirc github renders the markdown in a "context" relative to where the file sits -- so it knows that down one directory is the CONTRIBUTING.rst file (`./.github/../CONTRIBUTING.rst`)

Reply:
Maybe you use some browser extension?

Reply:
oh what now it doesn't work, what did I do before 😕

Reply:
maybe you used some different view (like from the actual file page, not PR)? Also, there's some front-end caching here.